.github/workflows/release_8_x_scheduled.yml

```diff
@@ -1,9 +1,5 @@
-# This workflow uses actions that are not certified by GitHub.
-# They are provided by a third-party and are governed by
-# separate terms of service, privacy policy, and support
-# documentation.
-# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
-# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+# If there are failures running this workflow, please resolve them on the release-8.x branch
+# and then port any necessary changes to this branch.
```

Comment:
Not a blocker, this comment just made me think that it could be potentially useful to have `on: workflow_dispatch` as another possible trigger.  This might help if you want to validate a fix you made to the `release-8.x` branch without waiting for the next scheduled time.

Reply:
Good idea, thanks!